### PR TITLE
Oscar/resources

### PR DESCRIPTION
**Pull request by Stereopoly**
Simulator Screen Shot - iPhone 11 Pro Max - 2020-03-12 at 19 01 12

- Resources page with data fetched from Firebase

### CODE REVIEW

**Comment by Stereopoly** [comment]
Backend is missing most data needed in the design.

**Comment by sidthesquid** [comment]
Nit: maybe set the inner card shadows to be basically the same as the outer container?

**Comment by Stereopoly** [comment]
Shadow now matches card. Maybe we should have some universal constants or smt so all the shadows are the same? @KevWho

**Comment by sidthesquid** [comment]
Yeah, I was thinking it would be similar to Shawn's PR. Should definitely have some universal views and constants. For example, the border radius of the cards, shadow, padding, etc. should be same across screens.

File: bm-persona/Map/MapViewController.swift [hunk 1]
```diff
@@ -142,7 +142,7 @@ extension MapViewController: SearchBarDelegate {
         guard !searchResultsView.isScrolling else { return }
         showSearchResultsView(false)
         searchBar.setButtonStates(hasInput: textField.text?.count != 0, isSearching: false)
-    }
+    }
```
Comment (KevWho) [inline bm-persona/Map/MapViewController.swift]: 🔥

File: bm-persona/Common/ResourceTableViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        get {
+            return super.frame
+        }
+        set (newFrame) {
+            var frame = newFrame
+            frame.origin.x += 5
+            frame.size.width -= 10
+            super.frame = frame
+        }
+    }
```
Comment (KevWho) [inline bm-persona/Common/ResourceTableViewCell.swift]: Same as I mentioned in Shawn's PR - may want to consider just using frame.inset

**Pull request merged**